healthtools_ke_api/build_query.py

```diff
@@ -101,131 +87,62 @@ def build_query_response(self, query):
             msg_items.append("1. Doctors: DR. SAMUEL AMAI")
             msg_items.append("2. Clinical Officers: CO SAMUEL AMAI")
             msg_items.append("3. Nurses: NURSE SAMUEL AMAI")
-            msg_items.append("4. NHIF accredited hospital: NHIF KITALE")
+            msg_items.append("4. NHIF accredited hospital: NHIF KITALE or NHIF KITALE INPATIENT")
             msg_items.append("5. Health Facility: HF KITALE")
             msg = " ".join(msg_items)
             return [msg, {'error': " ".join(msg_items)}]
+    
+    def construct_responses(self, docs_list):
+        hosy = ['nhif-outpatient-cs', 'nhif-outpatient','nhif-inpatient' ]
 
-    def construct_co_response(self, co_list):
-        # Just incase we found ourselves here with an empty list
-        if len(co_list) < 1:
-            return "Could not find a clinical officer with that name."
-        count = 1
-        msg_items = []
-        for co in co_list:
-            # co = co["fields"]
-            co = co["_source"]
-            status = " ".join(
-                [str(count) + ".", "".join(co['name'].title()), "-", "".join(co['qualifications'].upper())])
-            msg_items.append(status)
-            count = count + 1
-        if len(co_list) > 1:
-            msg_items.append(
-                "\nFind the full list at http://health.the-star.co.ke")
-        print "\n".join(msg_items)
-        return "\n".join(msg_items)
-
-    def construct_docs_response(self, docs_list):
-        # Just incase we found ourselves here with an empty list
         if len(docs_list) < 1:
-            return "Could not find a doctor with that name"
+            return "Could not find a anything with that name."
```

Comment:
This needs grammatical review. Also, we should definitely respond with what the user has requested. If a doctor, nurse, et al.